### PR TITLE
getUserCurrentDisputeStake

### PR DESCRIPTION
**Pull request by nuevoalex**

### CODE REVIEW

File: packages/augur-sdk/src/state/getter/Markets.ts [hunk 1]
```diff
@@ -1010,10 +1010,10 @@ function formatStakeDetails(db: DB, market: MarketData, stakeDetails: DisputeDoc
   for (let i = 0; i < stakeDetails.length; i++) {
     const outcomeDetails = stakeDetails[i];
     const outcomeValue = getOutcomeValue(market, outcomeDetails.payoutNumerators);
-    const bondSizeCurrent = new BigNumber(market.totalRepStakedInMarket, 16)
-    .multipliedBy(2)
-    .minus(new BigNumber(outcomeDetails.totalRepStakedInPayout || '0').multipliedBy(3)).toFixed();
-if (outcomeDetails.disputeRound < market.disputeRound) {
+    if (outcomeDetails.disputeRound < market.disputeRound) {
```
Comment (bthaile) [inline packages/augur-sdk/src/state/getter/Markets.ts:1016]: Is this a revert of my earlier PR?

File: packages/augur-sdk/src/state/getter/Markets.ts [hunk 2]
```diff
@@ -1028,7 +1028,7 @@ if (outcomeDetails.disputeRound < market.disputeRound) {
         outcome: outcomeValue.outcome,
         isInvalidOutcome: outcomeValue.invalid || false,
         isMalformedOutcome: outcomeValue.malformed || false,
-        bondSizeCurrent: new BigNumber(bondSizeCurrent).toFixed(),
+        bondSizeCurrent: new BigNumber(outcomeDetails.bondSizeCurrent || '0x0', 16).toFixed(),
```
Comment (bthaile) [inline packages/augur-sdk/src/state/getter/Markets.ts]: Did you figure out something upstream that caused (outcomeDetails.bondSizeCurrent to be undefined on newly disputed outcome in dispute round?

File: packages/augur-sdk/src/state/getter/Markets.ts (hunk 1 above)
Comment (nuevoalex) [reply packages/augur-sdk/src/state/getter/Markets.ts:1016]: Must have come from merging in master, will undo

File: packages/augur-sdk/src/state/getter/Markets.ts (hunk 2 above)
Comment (nuevoalex) [reply packages/augur-sdk/src/state/getter/Markets.ts]: Nope, just a bad merge!

**Review by bthaile — APPROVED** [review]
Lgtm, new ~~toy~~ getter to play with

**Pull request merged**